sqlglot/dialects/duckdb.py

```diff
@@ -957,15 +957,16 @@ def arraytostring_sql(self, expression: exp.ArrayToString) -> str:
         @unsupported_args("position", "occurrence")
         def regexpextract_sql(self, expression: exp.RegexpExtract) -> str:
             group = expression.args.get("group")
+            params = expression.args.get("parameters")
 
-            # Do not render group if it's the default value for this dialect
-            if group and group.name == str(self.dialect.REGEXP_EXTRACT_DEFAULT_GROUP):
+            # Do not render group if there is no following argument,
```

Comment:
oh, if you're in there changing things, one tiny improvement on the wording here could be
"We can skip group if there is no following argument". But not blocking

Reply:
Didn't find it hard to read in the first place but feel free to make any final touches, otherwise I'll go ahead and merge it soon